python/hail/dataset.py

```diff
@@ -4020,6 +4021,27 @@ def pc_relate(self, k, maf, block_size=512, min_kinship=-float("inf")):
         the threshold, then the variant's contribution to relatedness estimates
         is zero.
 
+        Under the PC-Relate model, kinship, \[ \phi_{ij} \], ranges from 0 to
+        0.5, and is precisely half of the
+        fraction-of-genetic-material-shared. Listed below are the statistics for
+        a few pairings:
+
+         - Monozygotic twins share all their genetic material so their kinship
+           statistic is 0.5 in expection.
+
+         - Parent-child and sibling pairs both have kinship 0.25 in expectation
+           and are separated by the identity-by-descent-zero, \[ k^{(2)}_{ij} \],
+           statistic which is zero for parent-child pairs and 0.25 for sibling
```

Comment:
@danking somehow it thinks you have non-ASCII characters here?

Reply:
Oh, yeah, these are non breaking spaces. I add them in so that math blocks don't span multiple lines. It's an old habit from other formats where new lines in math would break rendering. I'll remove them.